src/powershell/install-boxstarter.ps1

```diff
@@ -0,0 +1,43 @@
+# Install and configure Boxstarter
+# See: https://boxstarter.org/
+function Install-Boxstarter {
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://boxstarter.org/bootstrapper.ps1'))
+    Get-Boxstarter -Force
+
+    # Fix verbosity issues with Boxstarter v3
+    # See: https://github.com/chocolatey/boxstarter/issues/501
+    $fileToFix = "${Env:ProgramData}\boxstarter\boxstarter.chocolatey\Chocolatey.ps1"
+    $offendingString = 'if ($val -is [string] -or $val -is [boolean]) {'
+    if ((Get-Content $fileToFix -raw) -contains $offendingString) {
+        $fixString = 'if ($val -is [string] -or $val -is [boolean] -or $val -is [system.management.automation.actionpreference]) {'
+        ((Get-Content $fileToFix -raw) -replace [regex]::escape($offendingString),$fixString) | Set-Content $fileToFix
+    }
+    $fileToFix = "${Env:ProgramData}\boxstarter\boxstarter.chocolatey\invoke-chocolatey.ps1"
+    $offendingString = 'Verbose           = $VerbosePreference'
+    if ((Get-Content $fileToFix -raw) -contains $offendingString) {
+        $fixString = 'Verbose           = ($global:VerbosePreference -eq "Continue")'
+        ((Get-Content $fileToFix -raw) -replace [regex]::escape($offendingString),$fixString) | Set-Content $fileToFix
+    }
+}
+
+function Set-BoxstarterConfiguration {
+    $Boxstarter.RebootOk = (-not $noReboots.IsPresent)
+    $Boxstarter.AutoLogin = $true
+    $Boxstarter.SuppressLogging = $True
+    $global:VerbosePreference = "SilentlyContinue"
+    Set-BoxstarterConfig -NugetSources "$desktopPath;.;https://www.myget.org/F/vm-packages/api/v2;https://chocolatey.org/api/v2"
+    Set-WindowsExplorerOptions -EnableShowHiddenFilesFoldersDrives -EnableShowProtectedOSFiles -EnableShowFileExtensions -EnableShowFullPathInTitleBar
```

Comment:
Are these options specific to Boxstarter? Since they're modifying the behavior of Windows Explorer they seem more far-reaching. I just want to make sure they're being done in the most appropriate place if they are not exclusive to Boxstarter's needs.

Reply:
@mcdonnnj resolved in [dd989b3](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/dd989b3a9d9435e0e05bdfb2931bc46206bf1a84)

Reply:
@mcdonnnj yes this is a Boxstarter specific option. I added additional comments and a link to its doc in [4b0ef07](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/4b0ef07a29b9de24ef2b7c94a7edab347181c92f)

Reply:
I guess what I'm asking is are these Boxstarter specific (and required) configuration changes? The documentation you linked is just documenting the convenience commands Boxstarter provides for making Windows configuration changes. It does not say anything about Boxstarter needing these configuration changes to function.

Reply:
@mcdonnnj no this command is not required for Boxstarter to run properly, however, based on my research, its common for Boxstarter users accessing a vm to want these configuration changes set to quickly access various packages and navigate through through files and folders quickly. If you don't want them configured I can remove it. Just let me know.

Reply:
@itsmostafa If it's not required specifically for Boxstarter I have no issues leaving them but would prefer they are decoupled from the configuration of Boxstarter. Either as another script or at least a separate function to make it clear that we're using Boxstarter's convenience commands but it's not specifically a Boxstarter requirement.

Reply:
@mcdonnnj resolved in [15bf778](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/15bf778baf619975eb0fd8b4f8596a8fdcd8ca48)

Reply:
Suggested some tweaks but I'll close this conversation in favor of those since the meat of my request was covered.